### DISCUSSION TITLE
Tidak bisa terhubung dengan Perangkat

### DISCUSSION THREAD

**Discussion by ulilamri419** [kirimwa-id/forum#9] [Tanya Jawab] [answered]
Salam Mas Rio,

Saya ada kendala ketika menghubungkan dengan perangkat (Sesuai Step No.2):
* Berikut Request-nya:
<img width="632" alt="Task2 - Request Linked Device" src="https://user-images.githubusercontent.com/70904189/128721751-b1e25b99-6c45-4baa-9803-baeeca0c2474.png">

* Respone Http nya sudah keluar QR Code nya.
<img width="881" alt="Response QrCode" src="https://user-images.githubusercontent.com/70904189/128721832-5655e8e8-31b5-472b-b94e-70f798ae1b17.png">

* Kemudian saya copas Image_URL nya di browser sbb:
<img width="305" alt="Cuplikan QR-Code" src="https://user-images.githubusercontent.com/70904189/128722078-6105d885-5224-4647-9cd5-c97dae577927.png">

*Namun, ketika di Linked di Browser ke wa device, hasilnya Linked failed:
![Picture WA cannot Linked](https://user-images.githubusercontent.com/70904189/128721901-cacda97d-97e4-448b-ae6e-eae20c78d979.jpg)

Mohon pencerahannya, Terima kasih
Salam,
Amri

**Comment by rioastamal** [comment, marked as answer]
Halo mas @ulilamri419 ,

Tampilan code QR code memiliki masa waktu kurang lebih 20 detik semenjak response dari `/v1/qr`. 

1. Apakah scan dilakukan sebelum 20 detik?
2. Apakah tidak ada WhatsApp Web yang aktif ditempat lain?

**Reply by ulilamri419** [reply]
Halo Mas @rioastamal ,

Ternyata benar, ada limit waktu buat koneksi perangkat.. setelah saya coba secepat mungkin (under 10 detik), ternyata sudah connected status nya.

Terima kasih
BR
Amri
